platform/saivpp/vpplib/vppxlate/SaiVppXlate.c

```diff
@@ -1011,6 +1092,20 @@ static u32 get_swif_idx (vat_main_t *vam, const char *ifname)
     return ((u32) -1);
 }
 
+static const char * get_swif_name (vat_main_t *vam, const u32 swif_idx)
+{
+    hash_pair_t *p;
```

Comment:
Do you think a VPP_LOCK() is required while iterating over the list ?

Reply:
ok